ic-helpers/src/management/types.rs

```diff
@@ -0,0 +1,57 @@
+use candid::{CandidType, Deserialize, Principal};
+use ic_types::ic00::Payload;
+
+#[derive(CandidType, Deserialize, Debug, Clone, PartialEq, Eq, Hash)]
+pub struct Pubkey(Vec<u8>);
+
+impl std::default::Default for Pubkey {
+    fn default() -> Self {
+        Pubkey::empty()
+    }
+}
+
+impl Pubkey {
+    pub fn new(pubkey: Vec<u8>) -> Self {
+        Self(pubkey)
+    }
+
+    pub fn empty() -> Self {
+        Self(Vec::new())
+    }
+
+    pub fn as_bytes(&self) -> &[u8] {
+        &self.0
+    }
+}
+
+#[derive(CandidType, Deserialize, Debug)]
+pub struct GetECDSAPublicKeyArgs {
```

Comment:
Can't we use this structure from the `ic` dependency? Why do we duplicate it here?

Reply:
The same applies to other structures below

Reply:
They're removed with the current version of ic that we use